mindmaps-graql/src/main/java/io/mindmaps/graql/internal/analytics/DegreeAndPersistVertexProgram.java

```diff
@@ -113,7 +111,7 @@ public void workerIterationEnd(Memory memory) {
 
     @Override
     public boolean terminate(final Memory memory) {
```

Comment:
Out of curiosity, what is this method actually used for ? I have never seen a method called `terminate` that just does an equality check ?